src/AccessibilityInsights.SharedUx/FileBug/IssueReporterManager.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using AccessibilityInsights.SharedUx.Controls.SettingsTabs;
+using AccessibilityInsights.SharedUx.Settings;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+namespace AccessibilityInsights.SharedUx.FileBug
+{
+    /// <summary>
+    /// Class that deals with maintaining the available / selected issue reporters.
+    /// </summary>
+    public class IssueReporterManager
+    {
+        public static Guid SelectedIssueReporterGuid { get; set; }
+        readonly static object _lockObject = new object();
+        Dictionary<Guid, IIssueReporting> IssueReportingOptionsDict = new Dictionary<Guid, IIssueReporting>();
+        private static IssueReporterManager _defaultInstance = null;
+
+        public static IssueReporterManager GetInstance()
+        {
+            if (_defaultInstance == null)
+            {
+                lock (_lockObject)
+                {
+                    if (_defaultInstance == null)
+                    {
+                        _defaultInstance = new IssueReporterManager();
+                    }
+                }
+            }
+            return _defaultInstance;
+        }
+
+        private IssueReporterManager()
+        {
+            // Get all serialized configs
+            ConfigurationModel configs = ConfigurationManager.GetDefaultInstance().AppConfig;
+            var serializedConfigsDict = configs.IssueReporterSerializedConfigs;
+            Dictionary<Guid, string> configsDictionary = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigsDict);
```

Comment:
Hitting issues if `serializedConfigsDict` is `null.
```suggestion
            Dictionary<Guid, string> configsDictionary = new Dictionary<Guid, string>();
            if (serializedConfigsDict != null)
            {
                configsDictionary = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigsDict);
            }
```